todos/todo_list.py

```diff
@@ -0,0 +1,22 @@
+from .todo_item import ToDoItem
+
+
+class ToDoList:
```

Comment:
There is some mix of concerns with this class, it does both storage of the data and part of the presentation. Ideally we should have a in-memory data store that we can easily swap with a db implementation at a later time.

Reply:
Read about repository pattern and see if you can apply it here.